cmd/minishift/cmd/delete.go

```diff
@@ -68,6 +68,7 @@ func runDelete(cmd *cobra.Command, args []string) {
 	}
 
 	removeInstanceConfigs()
+	removeProfileFromConfig()
```

Comment:
this is tricky. Do we really remove the profile or mark is as some special state. After all we are not removing everything, do we? We might need to differentiate a bit here.

What if I delete profile 'bar' (after having made changes to say add-ons and persistent config), and then create a new 'bar' profile. Is this an error, or does it re-use the existing profile directory, re-using the existing files in there.

I think we might need an explicit `minishift profile delete [name]` which deletes the VM as we do here, but also deletes everything with it. The whole profile directory. A plain `minishift delete` is not deleting the profile.